### PR TITLE
chore: add store to schematics

### PR DESCRIPTION
**Pull request by raspberryredrose**
The `index.js` was auto-generated by running `yarn run build`.

### CODE REVIEW

File: src/store/files/__name@dasherize__/__name@dasherize__.selectors.ts.template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { createFeatureSelector } from '@ngrx/store';
+
+import { createEntityStoreSelectors } from '@shared/core/store/store.helpers';
+
+import { <%= classify(name) %>EntityAdapter, <%= classify(name) %>State } from './<%= dasherize(name) %>.state';
+
+export namespace <%= classify(name) %>Selectors {
+    export const select<%= classify(name) %>Vehicles = createFeatureSelector<<%= classify(name) %>State>('<%= classify(name) %>');
```
Comment (brunoalvesavrios) [inline src/store/files/__name@dasherize__/__name@dasherize__.selectors.ts.template:8]: I think Vehicles can be removed right?

File: src/store/files/__name@dasherize__/__name@dasherize__.selectors.ts.template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    export const {
+        selectItems,
+        selectLoading
+    } = createEntityStoreSelectors<T>(
+        <%= classify(name) %>EntityAdapter,
+        select<%= classify(name) %>Vehicles
```
Comment (brunoalvesavrios) [inline src/store/files/__name@dasherize__/__name@dasherize__.selectors.ts.template:15]: here as well?

File: src/store/files/__name@dasherize__/__name@dasherize__.selectors.ts.template (hunk 1 above)
Comment (raspberryredrose) [reply src/store/files/__name@dasherize__/__name@dasherize__.selectors.ts.template:8]: Good catch!

**Review by brunoalvesavrios — APPROVED** [review]
Nice to see how this is being done :]

**Pull request merged**